components/ExampleNav/ExampleNavContainer.coffee

```diff
@@ -4,6 +4,10 @@
 ExampleNav                     = require './ExampleNav'
 
 navs =
+  Dropdowns: [
+    'NavbarDropdownExamples'
```

Comment:
@reynard2007 I am more concerned when we merge this request after the one for generic dropdown component. And even though if it does not create conflicts, still this portion of code belongs to the othee task. So it would be better to have them separated for better tracking.